pkg/apis/componentconfig/types.go

```diff
@@ -346,3 +346,159 @@ type LeaderElectionConfiguration struct {
+	// cloudConfigFile is the path to the cloud provider configuration file.
+	// Empty string for no configuration file.
+	CloudConfigFile string `json:"cloudConfigFile"`
+	// concurrentEndpointSyncs is the number of endpoint syncing operations
+	// that will be done concurrently. Larger number = faster endpoint updating,
+	// but more CPU (and network) load.
+	ConcurrentEndpointSyncs int `json:"concurrentEndpointSyncs"`
+	// concurrentRSSyncs is the number of replica sets that are  allowed to sync
+	// concurrently. Larger number = more reponsive replica  management, but more
+	// CPU (and network) load.
+	ConcurrentRSSyncs int `json:"concurrentRSSyncs"`
+	// concurrentRCSyncs is the number of replication controllers that are
+	// allowed to sync concurrently. Larger number = more reponsive replica
+	// management, but more CPU (and network) load.
+	ConcurrentRCSyncs int `json:"concurrentRCSyncs"`
+	// concurrentResourceQuotaSyncs is the number of resource quotas that are
+	// allowed to sync concurrently. Larger number = more responsive quota
+	// management, but more CPU (and network) load.
+	ConcurrentResourceQuotaSyncs int `json:"concurrentResourceQuotaSyncs"`
+	// concurrentDeploymentSyncs is the number of deployment objects that are
+	// allowed to sync concurrently. Larger number = more reponsive deployments,
+	// but more CPU (and network) load.
+	ConcurrentDeploymentSyncs int `json:"concurrentDeploymentSyncs"`
+	// concurrentDaemonSetSyncs is the number of deployment objects that are
+	// allowed to sync concurrently. Larger number = more reponsive deployments,
+	// but more CPU (and network) load.
+	ConcurrentDaemonSetSyncs int `json:"concurrentDaemonSetSyncs"`
+	// concurrentJobSyncs is the number of deployment objects that are
+	// allowed to sync concurrently. Larger number = more reponsive deployments,
+	// but more CPU (and network) load.
+	ConcurrentJobSyncs int `json:"concurrentJobSyncs"`
+	// serviceSyncPeriod is the period for syncing services with their external
+	// load balancers.
+	ServiceSyncPeriod unversioned.Duration `json:"serviceSyncPeriod"`
+	// nodeSyncPeriod is the period for syncing nodes from cloudprovider. Longer
+	// periods will result in fewer calls to cloud provider, but may delay addition
+	// of new nodes to cluster.
+	NodeSyncPeriod unversioned.Duration `json:"nodeSyncPeriod"`
+	// resourceQuotaSyncPeriod is the period for syncing quota usage status
+	// in the system.
+	ResourceQuotaSyncPeriod unversioned.Duration `json:"resourceQuotaSyncPeriod"`
+	// namespaceSyncPeriod is the period for syncing namespace life-cycle
+	// updates.
+	NamespaceSyncPeriod unversioned.Duration `json:"namespaceSyncPeriod"`
+	// pvClaimBinderSyncPeriod is the period for syncing persistent volumes
+	// and persistent volume claims.
+	PVClaimBinderSyncPeriod unversioned.Duration `json:"pvClaimBinderSyncPeriod"`
+	// minResyncPeriod is the resync period in reflectors will be random between
```

Comment:
s/reflectors will/reflectors; will/
s/MinResyncPeriod and 2*MinResyncPeriod/minResyncPeriod and 2*minResyncPeriod/